continuous_integration/environment-3.7.yaml

```diff
@@ -3,32 +3,43 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - zstandard
-  - bokeh!=2.0.0
+  - python=3.7
+  - pip
+  - asyncssh
+  - bokeh
```

Comment:
It looks like the `!=2.0.0` was left off here. Not sure if that was intended. There are a few other environment files missing this detail as well.

Reply:
Thanks John. Not sure why this was dropped.